internal/cli/controls/login.go

```diff
@@ -0,0 +1,68 @@
+package controls
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	iHttp "iwakho/gopherkeep/internal/cli/http"
+	"iwakho/gopherkeep/internal/model"
+	"net/http"
+)
+
+var (
+	token string
+)
+
+type LoginCtrl struct{}
+
+func (LoginCtrl) Submit(p model.Pair) error {
+	r, err := http.NewRequest("GET", iHttp.ApiLogin, nil)
+	if err != nil {
+		return fmt.Errorf("Bad request: url=%s e=%s", iHttp.ApiLogin, err)
+	}
+	r.SetBasicAuth(p.Login, p.Password)
+	resp, err := iHttp.Client.Do(r)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("Bad status code: %d", resp.StatusCode)
+	}
+	token = resp.Header.Get("Authorization")
+	if token == "" {
+		return fmt.Errorf("Bad token")

```

Comment:
В соотвествии с нотацией Go текст ошибок нек должен модержать uppercase символы, если это не аббривеатуры.
```suggestion
		return fmt.Errorf("bad token")

```